Posts/meeting-23mar- gem updates

```diff
@@ -0,0 +1,376 @@
+---
+title: Raw IRC chat Mar 23rd 2016
+author: avsm (Anil Madhavapeddy)
+abstract: Raw IRC meeting notes on #mirage on Mar23 2016
+---
+
+```
+### Roll Call  
```

Comment:
Authors at the moment are just a plain string, you can put pretty much anything in there. Would it be useful to be able to find articles using the author's name ? i can change that if needed